versionedRouter.js

```diff
@@ -90,8 +91,12 @@ const functionsEnabled = () => {
 };
 
 async function handleDest(ctx, version, destination) {
-  const destHandler = getDestHandler(version, destination);
+  // const destHandler = getDestHandler(version, destination);
+  let destHandler;
   const events = ctx.request.body;
+  if (!events || !Array.isArray(events)) {
+    throw new Error("Event is missing or in inappropriate format", 400);
```

Comment:
`Error` class in Javascript doesn't have 2nd argument AFAIK, please change this to `CustomError`

Although this doesn't throw a runtime error because we used `Error` here but since it is an error thrown on transformer, the status code returned will be `5xx` only through the route to server, so it makes the 2nd argument(where `400` has been set) useless